client/my-sites/stats/summary/index.jsx

```diff
@@ -394,6 +394,7 @@ class StatsSummary extends Component {
 											path={ path }
 											period={ this.props.period }
 											skipQuery={ statType === 'statsUTM' }
+											hideIfNoData
```

Comment:
This causes many re-renders and repeat HTTP requests on the summary pages for fetching the stats

Reply:
It's about the lack of `skipQuery`, which causes the `QuerySiteStats` loop on the new navigation header.

https://github.com/Automattic/wp-calypso/blob/24ca61df6caeb3e9fbd9ff7ac34a3316d3bbc7fd/client/my-sites/stats/summary/index.jsx#L396

Reply:
You are right, I had actually disabled the query from everywhere but this got re-added when I was separating the PRs for location/UTM downloads - https://github.com/Automattic/wp-calypso/pull/103237/files#diff-0639383ff71d17e3cd6120bd4a34d1ace5e3860a8f811f9f95b373b0c7b4e2c7L387

Since we are now dispatching the data from the main stats cards into the download CSV buttons, the CSV download button does not need to make the requests

Reply:
This should be fixed now